frontend/components/modal/Error.vue

```diff
@@ -1,14 +1,46 @@
 <script setup lang="ts">
-const { error: { errorModalVisible, error } } = useModal()
+import { watch } from 'vue'
+
+interface Props {
+  isFullPage?: boolean
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  isFullPage: false
```

Comment:
IIUC, isFullPage is really just "is displaying over the error.vue page" right? If so maybe name it that?